src/screens/Grommet.js

```diff
@@ -113,7 +110,10 @@ export default () => (
           <PropertyValue type="string">
             <Description>
               'min' indicates it should take at minimum the whole viewport,
-              allowing the viewport to control scrolling.
+              allowing the viewport to control scrolling. This should be used in
```

Comment:
I think this is a little too narrow.

I think `true` should be used when the entire page layout will be controlled by a child Box or Grid, typically with a fixed header, footer, and/or sidebar, with scrolling being handled by the Main content within.

I think `"min"` should be used when the scrolling behavior includes all of the content on the page.